playpen/ansible/roles/db/tasks/main.yml

```diff
@@ -2,6 +2,7 @@
 - name: Install packages
   dnf: name={{ item }} state=present
   with_items:
+      - libselinux-python
```

Comment:
Ah, this is definitely an ansible requirement, like python-dnf. I''ve moved it to the bootstrap script.

Done